apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/logs/_components/DocumentLogs/DocumentLogInfo/Metadata.tsx

```diff
@@ -163,23 +167,33 @@ function DocumentLogParameters({
 }: {
   documentLog: DocumentLogWithMetadataAndError
 }) {
-  const parameters = useMemo(() => {
-    return Object.entries(documentLog.parameters).reduce(
-      (acc, [key, value]) => {
-        acc[key] = {
-          value: String(value),
-          metadata: { includeInPrompt: true },
-        }
-        return acc
-      },
-      {} as Inputs<'manual'>,
-    )
-  }, [documentLog.uuid, documentLog.parameters])
-
   return (
     <>
       <Text.H5M color='foreground'>Parameters</Text.H5M>
-      <InputParams inputs={parameters} disabled />
+      <div className='grid grid-cols-[auto_1fr] gap-y-3'>
+        {Object.entries(documentLog.parameters).map(
```

Comment:
could documentlog parameters be null or undefined?

Reply:
In theory no, but I will check for that just in case